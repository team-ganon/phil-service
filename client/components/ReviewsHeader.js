import React from 'react';
import style from '../CSS/reviewheader.css';

const ReviewsHeader = ({average, total}) => {
  return (
    <div className={style.reviewHeaderDiv}>
      <h2 className={style.reviewsHeaderH2}>Reviews</h2>
      <div className={style.bottomDivOfHeader}>
        <div className={style.containerForStarReviews}>
          <div className={style.star}>
            <ion-icon name="star"></ion-icon>
            <p className={style.averageTotal}>{average}</p>
          </div>
          <p className={style.average}><span className={style.reviewCount}>{total}</span> reviews</p>
        </div>   
        <div className={style.search}>
          <input type='text' placeholder='Search Reviews'/>
          <ion-icon name="search-outline"></ion-icon>
        </div>
      </div>
    </div>
  );
};

export default ReviewsHeader;