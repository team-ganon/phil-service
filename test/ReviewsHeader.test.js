import React from 'react';
import renderer from 'react-test-renderer';
import ReviewsHeader from '../client/components/ReviewsHeader';

describe('ReviewsHeader Component', () => {

  it('ReviewsHeader SnapShot Test', () => {
    const ReviewsHeaderJson = renderer.create(<ReviewsHeader average={4} total={100} />).toJSON();
    expect(ReviewsHeaderJson).toMatchSnapshot();
  });
  
});

