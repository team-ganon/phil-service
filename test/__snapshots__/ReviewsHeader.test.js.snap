// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewsHeader Component ReviewsHeader SnapShot Test 1`] = `
<div
  className="reviewHeaderDiv"
>
  <h2
    className="reviewsHeaderH2"
  >
    Reviews
  </h2>
  <div
    className="bottomDivOfHeader"
  >
    <div
      className="containerForStarReviews"
    >
      <div
        className="star"
      >
        <ion-icon
          name="star"
        />
        <p
          className="averageTotal"
        >
          4
        </p>
      </div>
      <p
        className="average"
      >
        <span
          className="reviewCount"
        >
          100
        </span>
         reviews
      </p>
    </div>
    <div
      className="search"
    >
      <input
        placeholder="Search Reviews"
        type="text"
      />
      <ion-icon
        name="search-outline"
      />
    </div>
  </div>
</div>
`;
